pdns/dnsdistdist/docs/reference/config.rst

```diff
@@ -1196,6 +1196,27 @@ Status, Statistics and More
 
   * ``outputFile=path``: string - Write the output of the command to the supplied file, instead of the standard output.
 
+.. function:: setStructuredLogging(enable[, options])
+
+  .. versionadded:: 1.9.0
+
+  Set whether log messages should be in a structured-logging-like format. This is turned off by default.
+  The resulting format looks like this (when timestamps are enabled via ``--log-timestamps`` and with ``levelPrefix="prio"`` and ``timeFormat="ISO8601"``)::
+
+  tm="2023-11-06T12:04:58+0100" prio="info" msg="Added downstream server 127.0.0.1:53"
```

Comment:
rec uses "ts", see https://docs.powerdns.com/recursor/appendices/structuredlogging.html
I 'd prefer if the keywords would match at least.
Also, the prio value are capitalized in rec. But this is debatable. The idea was to conform to other products, but we have not decided on the common terms. So this is good as a start in an alpha release, but it might need some changes in the future.

Reply:
> rec uses "ts", see https://docs.powerdns.com/recursor/appendices/structuredlogging.html I 'd prefer if the keywords would match at least.

It was my intent but I somehow confused myself, good catch!

> Also, the prio value are capitalized in rec. But this is debatable. 

Only for the `systemd-journal` backend, unless I'm mistaken? Do you think it would make sense for dnsdist to make it configurable as well?

> The idea was to conform to other products, but we have not decided on the common terms. So this is good as a start in an alpha release, but it might need some changes in the future.

Understood. I used https://github.com/PowerDNS/pdns/pull/13062 to determine the current common terms, which is still a work in progress. It would be nice to get together and decide on the common terms in the near future :)

Reply:
I mean initial cap. So "Info", not "info" or "INFO" in rec.

Reply:
My bad, I read `key` not `value`. I don't really have an opinion so I'll align on what the rec is doing.